packages/graphql/src/interfaces/graphqlServiceOption.interface.ts

```diff
@@ -40,4 +40,10 @@ export interface GraphqlServiceOption extends BackendServiceOption {
    * ex.: { field: "name", operator: EQ, value: "John" }
    */
   keepArgumentFieldDoubleQuotes?: boolean;
+
+  /**
+   * When false, searchTerms may be manipulated to be functional with certain filters eg: string only filters.
+   * When true, JSON.stringify is used on the searchTerms and used in the query "as-is". It is then the responsibility of the developer to sanitise the `searchTerms` property if necessary.
+   */
+  verbatimSearchTerms?: boolean;
```

Comment:
would it make more sense to rename this one to something that better represent a flag as opposed to the top one in the other interface which is a prop and not a flag. Maybe `useVerbatimSearchTerms`, we might need to rename `isWithCursor` to maybe `useCursor` (we could deprecate the old name with `@deprecate` until the next major). 

Let me know what you think, I'm just looking at these options and trying to align them and use better name if possible, I don't think anyone uses the `isWithCursor` yet since we just released it, that is from your work of last week, so now would be a good time to deprecate :)

oh and I guess you could also remove the text "(NOT FULLY IMPLEMENTED)" that I see in the comment up there.